src/data/sidebarData.js

```diff
@@ -6,7 +6,7 @@ export const sidebarSections = [
     title: 'Overview',
     icon: Globe,
     items: [
-      { id: 'how-it-works', title: 'What is FireWatch?' }
+      { id: 'how-it-works', title: 'Who it works?' }

```

Comment:
```suggestion
      { id: 'how-it-works', title: 'How it works?' }
```
Revisar la ortografía en ingles!!!!!